src/program.js

```diff
@@ -119,9 +119,14 @@ export class Program {
 
 
 export function defaultVersionGetter(absolutePackageDir: string): string {
-  let packageData: any = readFileSync(
+  if (process.env.NODE_ENV === 'production') {
+    let packageData: any = readFileSync(
```

Comment:
ah, right you are. It should be:

    let packageData: Buffer = readFileSync(...)

Let me know if you have any other questions about the feedback I left.